### PR TITLE
Feature/add cpg tests

### PR DESCRIPTION
**Pull request by michaelyoukeim**
- Fixed the failing tests in the CPG module
- Added a documentation

closes #1002

### CODE REVIEW

**Review by stschott — APPROVED** [review]

**Comment by stschott** [comment]
Closes #1002

**Review by swissiety — APPROVED** [review]
:+1:

**Comment by swissiety** [comment]
coverage went down a lot.. maybe exclude qilin from test reports at the moment - ill add a comment to the qilin+tests issue

File: sootup.report/pom.xml [hunk 1]
```diff
@@ -89,6 +89,18 @@
             <version>${project.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.soot-oss</groupId>
+            <artifactId>sootup.qilin</artifactId>
+            <version>${project.version}</version>
+            <scope>compile</scope>
+        </dependency>
```
Comment (swissiety) [inline sootup.report/pom.xml]:
```suggestion
```

**Comment by michaelyoukeim** [comment]
@swissiety Thanks! I'll merge with this status.

**Pull request merged**